pkg/k8s/deployments/translate_test.go

```diff
@@ -27,6 +27,7 @@ services:
   - name: worker
     container: dev
     image: worker:latest
+    localImage: true
```

Comment:
could we instead have an ImagePullPolicy attribute instead, to avoid introducing a new concept on the manifest?